files/en-us/web/css/@media/forced-colors/index.html

```diff
@@ -10,11 +10,7 @@
 ---
 <div>{{CSSRef}}</div>
 
-<p><span class="seoSummary">The <strong><code>forced-colors</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> <a href="/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Media_features">media feature</a> is used to detect if the {{Glossary("user agent")}} has enabled a forced colors mode where it enforces a user-chosen limited color palette on the page.</span></p>
-
-<div class="notecard note">
-<p><strong>Note:</strong> This feature is not supported by any user agent and its specifics are subject to change.</p>
-</div>
+<p><span class="seoSummary">The <strong><code>forced-colors</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> <a href="/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Media_features">media feature</a> is used to detect if the {{Glossary("user agent")}} has enabled a forced colors mode, such as Windows High Contrast, where it enforces a user-chosen limited color palette on the page.</span></p>
```

Comment:
```suggestion
<p><span class="seoSummary">The <strong><code>forced-colors</code></strong> <a href="/en-US/docs/Web/CSS">CSS</a> <a href="/en-US/docs/Web/CSS/Media_Queries/Using_media_queries#Media_features">media feature</a> is used to detect if the {{Glossary("user agent")}} has enabled a forced colors mode, where it enforces a user-chosen limited color palette on the page. An example of a forced colors mode is such as Windows High Contrast mode.</span></p>
```

Update this to simplify the sentence a bit.

Reply:
Done.